diff --git a/dist/index.mjs b/dist/index.mjs
index efc12cbd4f1e8f06dc724d8921b5d7d9c244de33..86ede674480faa03ec4214972aee26958da491f2 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -348,10 +348,10 @@ var EmojiFloatingGridBuilder = class {
 };
 
 // src/utils/EmojiLibrary/EmojiFloatingLibrary.ts
-import emojiMartData2 from "@emoji-mart/data";
+import emojiMartData2 from "@emoji-mart/data" with { type: "json" };
 
 // src/utils/EmojiLibrary/EmojiInlineLibrary.ts
-import emojiMartData from "@emoji-mart/data";
+import emojiMartData from "@emoji-mart/data" with { type: "json" };
 var EmojiInlineLibrary = class {
   constructor(library = emojiMartData) {
     this._hash = {};
