diff --git a/dist/index.mjs b/dist/index.mjs
index 3db72c864241a438a9ed3ea2860c311edcc2c410..63db96663e87e361b81c69ff535e19541427dcc5 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -312,10 +312,10 @@ var EmojiFloatingGridBuilder = class {
 };
 
 // src/utils/EmojiLibrary/EmojiFloatingLibrary.ts
-import emojiMartData2 from "@emoji-mart/data";
+import emojiMartData2 from '@emoji-mart/data' with { type: 'json' };
 
 // src/utils/EmojiLibrary/EmojiInlineLibrary.ts
-import emojiMartData from "@emoji-mart/data";
+import emojiMartData from '@emoji-mart/data' with { type: 'json' };
 var EmojiInlineLibrary = class {
   constructor(library = emojiMartData) {
     this._hash = {};
